packages/trader/src/Modules/Trading/Components/Form/TradeParams/Multiplier/info.tsx

```diff
@@ -54,7 +72,7 @@ const MultipliersInfo = observer(
             <Localize
                 i18n_default_text='<0>{{commission_percentage}}%</0> of (<1/> * {{multiplier}})'
                 values={{
-                    commission_percentage: Number((commission * 100) / (multiplier * amount)).toFixed(4),
+                    commission_percentage: ((Number(commission) * 100) / (multiplier * Number(amount))).toFixed(4),
```

Comment:
`commission` and `amount` can be `number | null | undefined`, so it make sense to cover them with `Number`, not the hole expression